solr_conf/conf/solrconfig.xml

```diff
@@ -11,10 +11,10 @@
   <directoryFactory name="DirectoryFactory" class="${solr.directoryFactory:solr.StandardDirectoryFactory}"/>
 
   <!-- solr lib dirs -->
```

Comment:
We do use the solrconfig.xml in production. And in the past we work with sysadmins to test these solrconfig files during solr upgrade in test/staging environment.